test_main.py

```
@@ -89,3 +158,70 @@ def test_save_markdown(mock_open):
 
     # assert write was called with the content
     mock_file.write.assert_called_once_with(markdown_content)
+
+
+def test_request_cache():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_request_cache`

Add return type annotation: `None`

(ANN201)